### ISSUE TITLE
"Not Found" error: can't save edits to an organization's repo

### ISSUE THREAD

**Issue by bijoutrouvaille** [prose/prose#898]
I wanted to set up a jekyll gh-pages repo that could be edited with prose by multiple users. For that I set up an organization and created the repo in it, myself being one of the admins. I'm experiencing two problems:

1. Repos which I'm authorized to edit but aren't part of my personal account don't show up in the list at prose.io/#myusername. This problem seems amenable by visiting prose.io/#organization-name/repo-name.
2. Here is the showstopper: I'm able to to edit files in step 1 above, but I can't save them. Here is a screenshot of the error I get when I click ‘commit:’

![image](https://cloud.githubusercontent.com/assets/897030/10971043/ccdd70f6-8397-11e5-8ef9-09472c56ef94.png)

If you don't have a quick fix or a workaround, I wonder if you could point me to the right area in the code so that I could attempt my own fix, if it's not too time-consuming.

#826 seems like it might be relevant.

**Comment by dereklieu** [comment]
@bijoutrouvaille have you tried removing your oAuth token for prose.io and re-authing with Github?

**Comment by bijoutrouvaille** [comment]
@dereklieu for the sake of due diligence I've re-authored the app, and I've noticed the "grant access" to the new org. Now I feel silly because I've gone through the process twice before, being aware of #826. 

What's strange, however, is that earlier I've made another github account to test this, and after I've re-authored my main account this last time (after your reply), the second account just started working without the need for re-authoring. I was using Chrome for the one account and FF for the other to avoid cross-pollution. Thanks.
